Makefile

```diff
@@ -0,0 +1,40 @@
+include .env
+
+CONTAINER_ENGINE ?= $(shell command -v podman 2> /dev/null || echo docker)
+COMPOSE_ENGINE ?= $(shell command -v podman-compose 2> /dev/null || echo docker-compose)
+MY_ID ?= $(shell id -u)
+CHANGE ?= New migration
+
+.PHONY: server
+
+# rebuild server image (for compose) with new and updated dependencies
+rebuild-server:
+	$(COMPOSE_ENGINE) build --no-cache server
+
+server-up:
+	$(COMPOSE_ENGINE) up --build --force-recreate -d server
+
+# run alembic revision in another pod
+alembic-generate-revision: server-up
+	@echo "Waiting for server to start and invoking alembic upgrade head..."
+	sleep 3
+
+	@echo "Checking if database is ready..."
+	$(CONTAINER_ENGINE) run --rm --network logdetective_default \
```

Comment:
I think `$(COMPOSE_ENGINE) exec postgres` should serve this purpose better

Reply:
I am not sure, in that way we will not check the "network connection", right? But if you think this is too complex I can also drop it. It only a bit useful to give a more meaningful error message in this case...

Reply:
if your proposed solution works, let's keep it :) I still didn't have time try it locally

sorry I did not explain what I had in mind precisely, I'll send followup PR once this is merged, that should make the explanation easier